pkg/reconciler/route/route_test.go

```diff
@@ -131,7 +131,7 @@ func newTestSetup(t *testing.T, opts ...reconcilerOption) (
 		la.Promote(reconciler.UniversalBucket(), func(reconciler.Bucket, types.NamespacedName) {})
 	}
 
-	return
+	return ctx, informers, ctrl, configMapWatcher, cf
```

Comment:
here's a place I could keep it :)

Reply:
I was close on this one too actually, but it's clearly "[more than a handful of lines](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)" so 🤷